README.md

```diff
@@ -38,6 +38,10 @@ yarn install
 yarn start
 ```
 
+You can also develop perf.html online in a pre-configured development environment:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/devtools-html/perf.html)
+
```

Comment:
can you please add a note in https://github.com/devtools-html/perf.html/blob/master/CONTRIBUTING.md#getting-started-with-development as well?